packages/connector-saml/src/index.test.ts

```diff
@@ -0,0 +1,86 @@
+import createConnector, { validateSamlAssertion } from './index.js';
+import { mockAttributes, mockedConfig } from './mock.js';
+
+const { jest } = import.meta;
+
+const getConfig = jest.fn().mockResolvedValue(mockedConfig);
+
+const setSession = jest.fn();
+const getSession = jest.fn();
+
+const connector = await createConnector({ getConfig });
+
+describe('getAuthorizationUri', () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('should get a valid uri and save required information to storage', async () => {
+    jest.useFakeTimers().setSystemTime(new Date('2023-01-01'));
```

Comment:
fake timers are seen in many place, wondering that are they really necessary?

Reply:
I've checked this. For the SAML assertion validation check, `fake timers` is needed since the check on issue time is mandated. And for UTs of `getAuthorizationUri`, we do not check the exact value of generated URI since the request ID is randomly generated. If the exact value of the URI needs to be checked, `fake timers` is also needed since the authentication requests include the timestamp.